sdk/src/main/scala/com/horizen/forge/Forger.scala

```diff
@@ -1,170 +1,138 @@
 package com.horizen.forge
 
+import java.time.Instant
+import java.util.concurrent.Executors
+
 import akka.actor.{Actor, ActorRef, ActorSystem, Props}
 import akka.pattern.ask
 import akka.util.Timeout
-import com.horizen.block.{MainchainBlockReference, SidechainBlock}
-import com.horizen.box.NoncedBox
+import com.horizen._
+import com.horizen.block.SidechainBlock
 import com.horizen.companion.SidechainTransactionsCompanion
-import com.horizen.consensus._
+import com.horizen.consensus.{ConsensusEpochNumber, ConsensusSlotNumber, TimeToEpochSlotConverter}
 import com.horizen.forge.Forger.ReceivableMessages.TryForgeNextBlockForEpochAndSlot
-import com.horizen.forge.Forger.SendMessages.{ForgeFailed, ForgeResult, ForgeSuccess, SkipSlot}
-import com.horizen.forge.Forger.View
 import com.horizen.params.NetworkParams
-import com.horizen.proposition.Proposition
-import com.horizen.secret.PrivateKey25519
-import com.horizen.transaction.SidechainTransaction
-import com.horizen.vrf.VRFProof
-import com.horizen.{ForgerDataWithSecrets, _}
-import scorex.core.NodeViewHolder.CurrentView
-import scorex.core.NodeViewHolder.ReceivableMessages.GetDataFromCurrentView
-import scorex.core.block.Block
-import scorex.util.{ModifierId, ScorexLogging}
+import scorex.core.NodeViewHolder.ReceivableMessages.LocallyGeneratedModifier
+import scorex.util.ScorexLogging
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success}
 
-
-case class ForgingInfo(parentId: Block.BlockId,
-                       timestamp: Block.Timestamp,
-                       mainchainBlockRefToInclude: Seq[MainchainBlockReference],
-                       txsToInclude: Seq[SidechainTransaction[Proposition, NoncedBox[Proposition]]],
-                       ownerPrivateKey: PrivateKey25519)
-
-
 class Forger(settings: SidechainSettings,
-             sidechainNodeViewHolderRef: ActorRef,
+             viewHolderRef: ActorRef,
              mainchainSynchronizer: MainchainSynchronizer,
              companion: SidechainTransactionsCompanion,
              val params: NetworkParams) extends Actor with ScorexLogging with TimeToEpochSlotConverter {
-
+  val forger: ForgeMessageBuilder = new ForgeMessageBuilder(mainchainSynchronizer, companion, params)
```

Comment:
`class Forger` contains value named `forger` that is actually `ForgeMessageBuilder`. Why not to name it `forgeMessageBuilder`?